02-Git/main.cpp

```diff
@@ -87,4 +89,15 @@ int main() {
             win();
         }
     }
-}
```

Comment:
Empty line is removed in last commit. please remove it first commit.
So this particular change will disappear.